src/main/kotlin/no/nav/familie/ba/sak/ekstern/pensjon/PensjonService.kt

```diff
@@ -82,18 +111,51 @@ class PensjonService(
             ?: error("Finner ikke tilkjent ytelse for behandling=${behandling.id}")
         return tilkjentYtelse.andelerTilkjentYtelse
             .filter { it.stønadTom.isSameOrAfter(fraDato.toYearMonth()) }
+            .filter { it.prosent != BigDecimal.ZERO && it.kalkulertUtbetalingsbeløp != 0 }
             .map {
                 BarnetrygdPeriode(
                     ytelseTypeEkstern = it.type.tilPensjonYtelsesType(),
                     personIdent = it.aktør.aktivFødselsnummer(),
                     stønadFom = it.stønadFom,
                     stønadTom = it.stønadTom,
-                    utbetaltPerMnd = it.kalkulertUtbetalingsbeløp,
-                    delingsprosentYtelse = it.prosent.toInt(),
+                    delingsprosentYtelse = when (it.prosent) {
+                        BigDecimal.valueOf(100L) -> YtelseProsent.FULL
+                        BigDecimal.valueOf(50L) -> YtelseProsent.DELT
+                        else -> YtelseProsent.USIKKER
```

Comment:
```suggestion
                        else -> YtelseProsent.DELT
```
Skal kanskje ikke være noen tvil om at den er delt 50/50 så lenge den ikke er full eller 0?